script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -26,19 +26,19 @@ No matter what I think, this is not an opponent that I can handle.
 "―――Arcueid, I――――"
 But is this really the right decision?
 Am I really going to abandon this woman?
-It's my fault. Am I really going to abandon this woman who can't even move properly because of me?
+It's my fault. She can't even move around properly because of me. Can I really turn my back on her like that?
 <Page8>
-"――――Am I――――"
-going to run away?
-Am I really going to run away from that monster that burned all those people ruthlessly and mercilessly without a care?
-Am I really going to pretend like I didn't see it, run away and survive alone without a shred of conscience?
+"――――I――――"
+Am I going to run away?
+Am I really going to run away from that ruthless monster, after they mercilessly burned away all those people like they were nothing?//興味もない being stretched a bit
```

Comment:
he rather than they. 'like they were nothing' works well. I can't think of a way to add an adverb without messing up the flow